R/get_draws_df.R

```diff
@@ -118,7 +157,7 @@ get_draws_df.data.frame <- function(x,
       lab = NA,
       site_pop = NA,
       below_lod = NA,
-      lod = NA,
+      log_lod = NA,
```

Comment:
I think that, if you haven't already, creating a function for validating input values would be good.

att @kaitejohnson

Reply:
You mean validating that the correct input values(e.g. rows of a column) are present? 

This is something we have (though the user can specify the name of their lod and conc column name, and then we change it to `log_lod` and `log_genome_copies_per_ml` https://github.com/CDCgov/ww-inference-model/blob/3a26b38e2c6e7365f8444f89f7db1e096826b372/R/checkers.R#L295

This will be modified a bit once the PRs are merged.